schema.graphql

```diff
@@ -819,6 +880,19 @@ input DeleteProjectInput {
   projectId: ID!
 }
 
+input WidgetLocationInput {
+  zone: String!
+  section: String!
+  area: String!
+}
+
+input WidgetLayoutInput {
+  extended: Boolean
+	location: WidgetLocationInput
+	index: Int
```

Comment:
```suggestion
  location: WidgetLocationInput
  index: Int
```

Reply:
On my local that extra space isn't showing... weird.